backend/init_db.py

```
@@ -105,6 +105,54 @@ def migrate_db():
                 # Index likely already exists
                 pass
 
+            # --- Grievance Migrations ---
+            # Add latitude column to grievances
+            try:
+                conn.execute(text("ALTER TABLE grievances ADD COLUMN latitude FLOAT"))
+                logger.info("Migrated database: Added latitude column to grievances.")
+            except Exception:
+                pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)